platformio.ini

```diff
@@ -16,51 +16,109 @@ framework = arduino
 [env:flight]
 build_flags = -DIP_ADDRESS_END=42
 src_filter = -<*> +<Common.h> +<Comms.*> +<FC/*>
+lib_deps = 
+	stevemarple/MicroNMEA@^2.0.5
+	sparkfun/SparkFun u-blox Arduino Library@^1.8.11
+	sparkfun/SparkFun u-blox GNSS Arduino Library@^2.2.0
```

Comment:
For these dependencies, can you download them and put them into the `lib` folder? That way we don't need to have external dependencies